gcp_variant_transforms/libs/vcf_header_parser.py

```diff
@@ -42,7 +42,7 @@ def get_vcf_headers(input_file):
   try:
     vcf_reader = vcf.Reader(fsock=_line_generator(input_file))
   except (SyntaxError, StopIteration) as e:
-    raise ValueError('Invalid VCF header: %s' % str(e))
+      raise ValueError('Invalid VCF header in %s: %s' % (input_file, str(e)))
```

Comment:
Sorry, my github suggestion somehow added two additional spaces and our linter is complaining about them. Could you please remove the two spaces just before `raise`?